### ISSUE TITLE
Building project results in cannot read property type of undefined

### ISSUE THREAD

**Issue by jsacapdev** [Microsoft/vscode-azurefunctions#412]
*** Steps to reproduce ***

1. Open VS Code Insiders
2. Open workbench.view.extension.azure (left)
3. Create new project
4. Select Folder
5. Choose C#
6. Choose option to Open in current Window
7. Build results in `cannot read property type of undefined`

This has only happened recently. 

`dotnet restore` & `dotnet build` works on the command line.

```
Version 1.25.0-insider
Commit 2040792965d90825a47aa1986cff8eb14a34b28e
Date 2018-06-07T05:16:56.305Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```

Extension: 0.9.1

tasks.json:

```
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "clean",
      "command": "dotnet clean",
      "type": "shell",
      "presentation": {
        "reveal": "always"
      },
      "problemMatcher": "$msCompile"
    },
    {
      "label": "build",
      "command": "dotnet build",
      "type": "shell",
      "dependsOn": "clean",
      "group": {
        "kind": "build",
        "isDefault": true
      },
      "presentation": {
        "reveal": "always"
      },
      "problemMatcher": "$msCompile"
    },
    {
      "label": "Run Functions Host",
      "identifier": "runFunctionsHost",
      "type": "shell",
      "dependsOn": "build",
      "options": {
        "cwd": "${workspaceFolder}/bin/Debug/netstandard2.0"
      },
      "command": "func host start",
      "isBackground": true,
      "presentation": {
        "reveal": "always"
      },
      "problemMatcher": []
    }
  ]
}

```

launch:

```
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Attach to C# Functions",
      "type": "coreclr",
      "request": "attach",
      "processId": "${command:azureFunctions.pickProcess}"
    }
  ]
}
```

**Comment by jsacapdev** [comment]
Is there an update on this? At the moment the extension is unusable.

**Comment by nturinski** [comment]
Hi @jsacapdev 

Sorry to hear about your experience.  I was having issues reproing the bug until I updated and it seems that updating VS Code Insiders to 1.25.0-insiders is what broke the extension.

I don't want you to be blocked, so while it's not an ideal solution, I've confirmed that the latest VS Code (1.24.0) still locally debugs the functions.

Sorry again for the inconvenience.  Hopefully, this will be resolved soon.
